test_pytest_rerunfailures.py

```diff
@@ -15,263 +16,326 @@ def temporary_failure(count=1):
             if int(count) <= {0}:
                 path.write(int(count) + 1)
                 raise Exception('Failure: {{0}}'.format(count))""".format(
-        count)
+        count
+    )
 
 
 def check_outcome_field(outcomes, field_name, expected_value):
     field_value = outcomes.get(field_name, 0)
-    assert field_value == expected_value, \
-            "outcomes.{} has unexpected value. Expected '{}' but got '{}'" \
-            .format(field_name, expected_value, field_value)
+    assert (
+        field_value == expected_value
+    ), "outcomes.{} has unexpected value. Expected '{}' but got '{}'".format(
+        field_name, expected_value, field_value
+    )
 
 
-def assert_outcomes(result, passed=1, skipped=0, failed=0, error=0, xfailed=0,
-                    xpassed=0, rerun=0):
+def assert_outcomes(
+    result, passed=1, skipped=0, failed=0, error=0, xfailed=0, xpassed=0, rerun=0,
+):
     outcomes = result.parseoutcomes()
-    check_outcome_field(outcomes, 'passed', passed)
-    check_outcome_field(outcomes, 'skipped', skipped)
-    check_outcome_field(outcomes, 'failed', failed)
-    check_outcome_field(outcomes, 'xfailed', xfailed)
-    check_outcome_field(outcomes, 'xpassed', xpassed)
-    check_outcome_field(outcomes, 'rerun', rerun)
+    check_outcome_field(outcomes, "passed", passed)
+    check_outcome_field(outcomes, "skipped", skipped)
+    check_outcome_field(outcomes, "failed", failed)
+    check_outcome_field(outcomes, "xfailed", xfailed)
+    check_outcome_field(outcomes, "xpassed", xpassed)
+    check_outcome_field(outcomes, "rerun", rerun)
 
 
 def test_error_when_run_with_pdb(testdir):
-    testdir.makepyfile('def test_pass(): pass')
-    result = testdir.runpytest('--reruns', '1', '--pdb')
-    result.stderr.fnmatch_lines_random(
-        'ERROR: --reruns incompatible with --pdb')
+    testdir.makepyfile("def test_pass(): pass")
+    result = testdir.runpytest("--reruns", "1", "--pdb")
+    result.stderr.fnmatch_lines_random("ERROR: --reruns incompatible with --pdb")
 
 
 def test_no_rerun_on_pass(testdir):
-    testdir.makepyfile('def test_pass(): pass')
-    result = testdir.runpytest('--reruns', '1')
+    testdir.makepyfile("def test_pass(): pass")
+    result = testdir.runpytest("--reruns", "1")
     assert_outcomes(result)
 
 
 def test_no_rerun_on_skipif_mark(testdir):
     reason = str(random.random())
-    testdir.makepyfile("""
+    testdir.makepyfile(
+        """
         import pytest
-        @pytest.mark.skipif(reason='{0}')
+        @pytest.mark.skipif(reason='{}')
         def test_skip():
             pass
-    """.format(reason))
-    result = testdir.runpytest('--reruns', '1')
+    """.format(
+            reason
```

Comment:
Those formatings are somehow not so easy to read. Looking forward to f-strings in this case when we stop Supporting Python 2.

And I really do not understand, why this happens. This function call could easily fit on one line. But that is the way you work with black, I guess.

Reply:
> Looking forward to f-strings in this case when we stop Supporting Python 2.

What's stopping you? :wink: FWIW you'd also need to drop 3.5 for f-strings, which is currently still [supported upstream](https://devguide.python.org/#status-of-python-branches) for another 2 months (but dropped in various projects already)

Reply:
That is true, but was not part of the formatting PR.

Reply:
And I really do not understand, why this happens. This function call could easily fit on one line. But that is the way you work with black, I guess.

I tried to reformat the calls, but black still reformatted them. Are you ok with it staying as is?

Reply:
Let's keep it as it is. This is what happens when using automated tools. The goal when using black was to avoid such formatting discussions in the first place.

Reply:
Yeah, this will go away when we use f-strings. We will use black as it is now.